openid-federation-1_0.xml

```diff
@@ -1219,6 +1222,14 @@
           All Federation Entities MAY publish a
 	      <spanx style="verb">federation_historical_keys_endpoint</spanx>.
 	    </t>
+	    <t hangText="entity_statement_signing_alg_values_supported">
+	      <vspace/>
+	      OPTIONAL.
+	      JSON array containing a list of the JWS signing algorithms
+	      (<spanx style="verb">alg</spanx> values)
+	      supported by the Entity to sign Entity Statements.
+	      If omitted, the default value is <spanx style="verb">["RS256"]</spanx>.
```

Comment:
On its own, the definition of this parameter is okay. However, having this parameter parameter defined would not be sufficient for the entities in a federation to be able to use a specific JWS algorithm when the more than one is supported. People will ask - what do we do then with this parameter? How do I get an ES signed with say ES256 when RS256 and ES256 are supported?